test/puppeteer.js

```diff
@@ -2,7 +2,7 @@ const puppeteer = require('puppeteer');
 
 (async () => {
 	const browser = await puppeteer.launch();
-	const page = await browser.newPage();
+	//const page = await browser.newPage();
 	// extend
 	await browser.close();
-})().catch(err => console.log(err));
+})().catch(err => err);
```

Comment:
Can we get rid of this `catch` instead of silently ignoring the error? If there was a problem we want this to shout and let us to know what's wrong